README.md

```diff
@@ -50,5 +50,6 @@ bug in Closure Compiler).
 ## Supported Version of TypeScript
 Clutz produces declaration files that are guaranteed to be accepted by a
 version of TypeScript
-[2.0.6](https://github.com/Microsoft/TypeScript/tree/v2.0.6). The current test
-suite runs against `typescript@next`, so that is always a good choice.
+[2.1.6](https://github.com/Microsoft/TypeScript/tree/v2.1.6). The current test
+suite runs against the version of `typescript` in `npm-shrinkwrap.json`, so that is
```

Comment:
Travis supports only a single language per project. We have 'java'. In java environments npm seems to be present but not yarn. I tried to install yarn in the before_script, but it failed so I gave up on it for now.

https://travis-ci.org/angular/clutz/builds/204840916